.gitignore

```diff
@@ -8,3 +8,9 @@ sonica_pb2_grpc.py
 sonica_pb2.py
 sonica.pb.cr
 sonica_services.pb.cr
+
+# Files created when installing crystal and shards locally
+bin/
+lib/
+shard.lock
+shard.yml
```

Comment:
These should be path-specific, which i think is done with `/shard.lock` to say it's only the one next to the .gitignore file.
Otherwise we can't use crystal in any part of sonica ^^

Reply:
Should these all be prepended with `/` then?

Reply:
If that is what makes the paths relative, because `shard.lock` will match any file with that name in the repo.
I don't remember the correct way of doing it.